### DISCUSSION TITLE
ScriptReference/PlayerPrefs.GetString

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#539] [Notes]
# ScriptReference/PlayerPrefs.GetString

https://docs.unity3d.com/ScriptReference/PlayerPrefs.GetString.html

**Comment by PeturDarri** [comment]
Passing `null` as the `defaultValue` will instead return an empty string when the key doesn't exist.

**Reply by PeturDarri** [reply]
More about it here: https://forum.unity.com/threads/solved-how-to-tell-if-playerprefs-is-null.485789/#post-3746371
